### ISSUE TITLE
Bug - Repeated sequence in pattern

### ISSUE THREAD

**Issue by cgsecurity** [longld/peda#70]
```
gdb-peda$ pattern_create 179
'AAA%AAsAABAA$AAnAACAA-AA(AADAA;AA)AAEAAaAA0AAFAAbAA1AAGAAcAA2AAHAAdAA3AAIAAeAA4AAJAAfAA5AAKAAgAA6AALAAhAA7AAMAAiAA8AANAAjAA9AAOAAkAAPAAlAAQAAmAARAAnAASAAoAATAApAAUAAqAAVAArAAWAAsA'

```
'AAsA' can found twice (offset 5 and at the end)
The problem may be in de_bruijn() implementation

**Comment by longld** [comment]
Fixed in 1bb9995

**Comment by longld** [comment]
Closed. Thanks.
